### DISCUSSION TITLE
Remove support for netstandard2.0

### DISCUSSION THREAD

**Discussion by AskBojesen** [dotnet-bluetooth-le/dotnet-bluetooth-le#744] [General]
When netstandard2.0 is included as target framework, then we are bound to use C# 7.3

Either upgrade to netstandard2.1 to be able to use C# 8.0 or remove support for netstandard

Ref https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version

**Comment by janusw** [comment]
I think Xamarin.UWP is not compatible with netstandard2.1 (don't ask me why). So this will only happen once we deprecate Xamarin, I guess. Since Xamarin is still supported by MS until May 2024 (see https://dotnet.microsoft.com/en-us/platform/support/policy/xamarin), I would like to keep supporting it in the library until that date as well.

Is there a particular C# 8 feature you need?

**Reply by AskBojesen** [reply]
No, just a few habits with new C# language syntax features we use in our other code.

Just a suggestion, keep netstandard2 if you want to - not blocking anything for us.
